packages/authentication/README.md

```diff
@@ -20,18 +20,27 @@ npm install --save @loopback/authentication
 Start by decorating your controller methods with `@authenticate` to require
 the request to be authenticated.
 
+In this example, we make the user profile available via dependency injection
+using a key available from `@loopback/authentication` package.
+
 ```ts
-// controllers/my-controller.ts
-import {UserProfile, authenticate, AuthenticationBindings} from '@loopback/authentication';
-import {inject} from '@loopback/core';
-import {get} from '@loopback/rest';
+// src/controllers/who-am-i.controller.ts
+import {inject} from '@loopback/context';
+import {
+  AuthenticationBindings,
+  UserProfile,
+  authenticate,
+} from '@loopback/authentication';
+import {get} from '@loopback/openapi-v3';
```

Comment:
We should be advocating import of `get` from `@loopback/rest` imo as it's one less package a user has to learn / understand.

Reply:
@virkt25 Honestly, I haven't given much thoughts to where users should be importing functions that are shared by multiple modules; I personally just took copy pasted the original code without the imports and then imported the first thing that vscode auto-completed for me, but I do think it's relevant for us to be consistent with ourselves on where to import functions like these from (same thing for your comment after this). I'm going to look for a preexisting issue to see if we have this tracked, if not, I'm going to create one. For now, I want to leave these alone and wait until we decide on the consistency of these imports.

Reply:
Sounds good. Once you have an issue to track this, I can dismiss these comments.

Reply:
I'm on the fence on whether or not we should re-export these decorators in `@loopback/rest`, but sounds good.

Reply:
`get` along with `post`, `put`, etc. are already re-exported from `@loopback/rest` and should be used from there IMO

Reply:
Issue's been created here https://github.com/strongloop/loopback-next/issues/1117

Reply:
Please switch `get` to be from `@loopback/rest` for this PR and if something else is decided in #1117, we can update it in another PR.